### DISCUSSION TITLE
Bug triage/handling?

### DISCUSSION THREAD

**Discussion by mimre25** [regolith-linux/regolith-desktop#990] [Q&A] [answered]
Hi, I left a bug description a couple weeks ago in https://github.com/regolith-linux/regolith-desktop/issues/985 but haven't seen any triaging or response yet. 

Have I done something wrong, or what would be the best way to get my issues addressed?

**Comment by kgilmer** [comment, marked as answer]
Hi @mimre25 , no you haven't done anything wrong.  Regolith is a project made by volunteers.  Typically if I don't have anything actionable to suggest on a ticket I let it be in hopes that someone else has had a similar experience.
